dashboard/app/views/levels/_single_multi.html.haml

```diff
@@ -119,7 +119,7 @@
 
   :ruby
     js_data = {
-      response_count: @responses&.count || 0,
+      response_count: @responses&.first&.count || 0,
```

Comment:
Why is this being changed?

Reply:
This is changed because the structure of `@responses` is now an array of responses objects as opposed to just a single object, so this adds a safe call in case the array is empty